### ISSUE TITLE
Add basic functional "smoke test"

### ISSUE THREAD

**Issue by adhilto** [cisagov/ScubaGoggles#171]
# 💡 Summary #
Add a functional, end-to-end test that performs very basic validation of ScubaGoggles functionality.

## Motivation and context ##
Currently all functional testing needs to be done manually, leading to delayed discovery of potential issues. Due to GWS's limited API, we can't implement full functional testing, but we could implement a basic "smoke test" which would allow us to quickly identify some issues.

## Implementation notes ##
Dependent on:
- https://github.com/cisagov/ScubaGoggles/pull/165
- https://github.com/cisagov/ScubaGoggles/pull/163

After those two PRs are merged, we can add a GitHub trigger that runs ScubaGoggles and performs basic validation of the report, such as:
- The provider output is valid json
- There is no missing output
- No API calls or functions produced exceptions

I recommend referring to the ScubaGear smoke test as a starting point for implementing this functionality.

See:
- https://github.com/cisagov/ScubaGear/blob/main/.github/workflows/run_smoke_test.yaml
- https://github.com/cisagov/ScubaGear/tree/main/Testing/Functional/SmokeTest

## Acceptance criteria ##
- [ ] A basic smoke test has been implemented
- [ ] The smoke test has been added to the GitHub workflow

**Comment by LaurenBassett** [comment]
Just waiting on error handling PR #163

**Comment by adhilto** [comment]
Both dependent PRs have been merged into main, this issue is ready for development.
